features/gammatone.py

```diff
@@ -29,27 +33,53 @@ def GammatoneJob(crp, gt_options=None, **kwargs):
 
 
 def gammatone_flow(
-    minfreq=100,
-    maxfreq=7500,
-    channels=68,
+    minfreq: int = 100,
+    maxfreq: int = 7500,
+    channels: int = 68,
     warp_freqbreak=None,
```

Comment:
according to https://github.com/rwth-i6/rasr/blob/907eec4f4e36c11153f6ab6b5dd7675116f909f6/src/Signal/GammaTone.cc#L228-L234 these can be of type float. But we do anyway always use ints here.. so no strong opinion on this one 🤷 

```suggestion
    minfreq: float = 100,
    maxfreq: float = 7500,
    channels: int = 68,
    warp_freqbreak: Optional[float] = None,
```

Reply:
I think we usually treat frequencies as integers, so I would just leave it. If a person ever needs floats here that person would have so much knowledge about the pipeline that typing hints would not be a hindrance anyway :)